internal/fs/fs.go

```diff
@@ -1884,6 +1901,16 @@ func (fs *fileSystem) Unlink(
 	parent.Lock()
 	defer parent.Unlock()
 
+	inodeName := inode.NewFileName(parent.Name(), op.Name)
+	fs.mu.Lock()
+	_, ok := fs.localFileInodes[inodeName]
```

Comment:
delete it from fs.inodes as well.

Reply:
The ForgetInode() call will ensure the deletion from fs.inodes